variant_calling_benchmarks/joint_caller/process_results.py

```
@@ -53,12 +55,17 @@ def merge_calls_with_others(config, guacamole_calls_df):
     merged = guacamole_calls_df
 
     for (name, info) in config['variants'].items():
-        df = pandas.read_csv(info.get_substituted('path', path=True))
+        variant_file = info.get_substituted('path', path=True)
+        df = load_benchmark_variants(variant_file)
 
         # Since we load guacamole VCFs with varcode, the contigs will be
         # normalized, so we have to normalize them here.
         df["contig"] = df.contig.map(normalize_chromosome)
         df["called_%s" % name] = True
+        join_columns = JOIN_COLUMNS
+        if 'patient' in df.columns:
```

I think we may want to just throw an error here if there are >1 patients in the benchmark and `df` does not have a 'patient' column. Hard to imagine doing anything reasonable in that case